cpp/oneapi/dal/detail/vertex_partitioning_ops.hpp

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include "oneapi/dal/detail/policy.hpp"
+
+namespace oneapi::dal::preview {

```

Comment:
It's better to combine namespaces in single line

Reply:
In other *_ops.hpp files namespaces are split into 2 lines
For example https://github.com/oneapi-src/oneDAL/blob/master/cpp/oneapi/dal/detail/traverse_ops.hpp#L21